code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -330,3 +330,19 @@
 	w_class = ITEMSIZE_NORMAL
 	can_hold = list(/obj/item/book, /obj/item/spellbook)
 
+	// -----------------------------
+	//           Chemistry Bag
+	// -----------------------------
+/obj/item/storage/bag/chemistry
+	name = "chemistry bag"
+	icon = 'icons/obj/chemical.dmi'
+	item_icons = list(
+		slot_l_hand_str = 'icons/mob/items/lefthand_medical.dmi',
+		slot_r_hand_str = 'icons/mob/items/righthand_medical.dmi',
+		)
+	icon_state = "chembag"
+	desc = "A bag for storing pills, patches, and bottles."
```

Comment:
Aurora doesn't use medical patches, would be best to remove that bit from the item description.